client/components/card/style.scss

```diff
@@ -91,7 +91,8 @@ a.card:hover,
 
 a.card:focus,
 .is-card-link.card:focus {
-	outline: 0;
+	outline: 2px solid var( --color-primary-light );
+	z-index: 1;
```

Comment:
this will apply to every single instance of `card`. Is that what you want? Why do we need a z-index?

Reply:
It was @enriquesanchez who introduced the outline change... I also share your concerns.

The z-index is needed as the outline sits underneath the next card in the list without it, so it looks unbalanced.